src/main/java/io/personium/common/auth/token/AbstractLocalToken.java

```diff
@@ -228,41 +242,41 @@ public static String encode(final String in, final byte[] ivBytes) {
     }
 
     /**
-     * 復号する.
-     * @param in 暗号化文字列
-     * @param ivBytes イニシャルベクトル
-     * @return 復号された文字列
-     * @throws AbstractOAuth2Token.TokenParseException 例外
+     * decode a ciphered string using an initial vector.
+     * @param in ciphered string
+     * @param ivBytes Initial Vector bytes
+     * @return decoded string
+     * @throws AbstractOAuth2Token.TokenParseException
      */
     public static String decode(final String in, final byte[] ivBytes) throws AbstractOAuth2Token.TokenParseException {
         byte[] inBytes = CommonUtils.decodeBase64Url(in);
         Cipher cipher;
         try {
             cipher = Cipher.getInstance(AES_CBC_PKCS5_PADDING);
         } catch (NoSuchAlgorithmException e) {
-            throw AbstractOAuth2Token.PARSE_EXCEPTION;
+            throw new TokenParseException(e);
```

Comment:
All error handling is same. So put together them into one try-catch block.

```java
        try {
            cipher = Cipher.getInstance(AES_CBC_PKCS5_PADDING);
            cipher.init(Cipher.DECRYPT_MODE, aesKey, new IvParameterSpec(ivBytes));
            byte[] plainBytes;
            plainBytes = cipher.doFinal(inBytes);
            return new String(plainBytes, CharEncoding.UTF_8);
        } catch (Exception e) {
            throw new TokenParseException(e);
        }
```

Reply:
Thanks